README.md

```diff
@@ -21,11 +27,26 @@ API documentation can be found at https://mugi-backend.onrender.com/docs
 - Python is installed
 - Poetry is installed
 - PostgreSQL is installed
+- This repo has been cloned locally
+  - Firebase-adminsdk `.json` file has been obtained separately from a team member and added into the root directory of your local copy of this repo
+
+### Initial Configuration
+
+1. Using `psql`, run `CREATE DATABASE petsitter;` to make the database
+
+2. Create a `.env` file in root directory of your local copy of this repo
+
+3. In your `.env` file, add the following three variables and assign their values as follows:
+  - `DATABASE_URL`: Value should be the path specific to your local database (starting with `postgres://`)
+  
+  - `FIREBASE_CREDENTIALS_PATH`: Value should be the exact file name of the firebase-adminsdk `.json` file referenced above
```

Comment:
Since I changed this part and we are not using Firebase-adminsdk `.json` file, could you change it to something like this. 

FIREBASE_CREDENTIALS : Value should be a base64 encoded credential file